lib/sparql_to_sw_solr/instance_solr_doc.rb

```diff
@@ -1,8 +1,16 @@
+require 'sparql/client'
+require 'linkeddata'
+
 module SparqlToSwSolr
   class InstanceSolrDoc
 
-    # TODO: get this from settings.yml
+    # TODO: get these from settings.yml
     BASE_URI = 'http://ld4p-test.stanford.edu/'.freeze
+    BF_NS = 'http://id.loc.gov/ontologies/bibframe/'.freeze
+    BF_NS_DECL = "PREFIX bf: <#{BF_NS}>".freeze
+    SPARQL_URL = 'http://sul-ld4p-blazegraph-dev.stanford.edu/blazegraph/namespace/ld4p/sparql'.freeze
```

Comment:
We should go with "localhost" and with 8080, tbh;  it should be a more direct way to reference the same endpoint.  The experiments were all about me trying to figure out how to use the endpoint.